examples/command_center/command_center.ino

```diff
@@ -4,101 +4,63 @@
 // This example code shows how to use an Adafruit Feather M0 module and Microsoft Azure for IoT
 // applications.
 
-#include <stdlib.h>
-#include <stdio.h>
-#include <stdint.h>
-#include <time.h>
-#include <sys/time.h>
-#include <SPI.h>
-#ifdef ARDUINO_SAMD_FEATHER_M0
+// change the next three lines to use on non-Adafruit WINC1500 based boards/shields
```

Comment:
Using C macros is something we should avoid. It's not "Arduino" style. The [Arduino Style Guide for Writing Libraries](http://www.arduino.cc/en/Reference/APIStyleGuide) is a good read, most of the Arduino user base is not familiar with them.

Also, once https://github.com/arduino-libraries/WiFi101/issues/57 is resolved by something like https://github.com/arduino-libraries/WiFi101/pull/58 we can simplify these quite a bit.